code/style.css

```diff
@@ -1,26 +1,57 @@
 /* Global css variables used for colors */
+
 :root {
-  --primary: #a259ff;
-  --secondary: #b0a6ff;
+  --primary: #a0cdd4;
+  --secondary: #af8eae;
 }
 
 body {
   display: flex;
   margin: 0;
   padding: 0;
-  font-family: 'Montserrat', sans-serif;
+  font-family: 'Italiana', serif;
+}
+
+.timer-guess-count {
+  display: flex;
+  flex-direction: row;
+}
+
+.timer, .guess-count {
+  display: flex;
+  justify-content: center;
+  border: var(--primary) solid;
+  background-color: var(--primary);
+  border-radius: 35px;
+  padding: 0 9%;
+  font-size: 22px;
+  letter-spacing: 1px;
+  color: white;
+  width: 110px;
+  margin-bottom: 5px;
+  font-family: 'Italiana', serif;
+  font-weight: bold;
```

Comment:
if you add a width: 80% it would stop the time div from shrinking and expanding all the time (pun-intended again)